pkg/beacon/relay/gjkr/protocol.go

```diff
@@ -549,7 +549,8 @@ func (sjm *SharesJustifyingMember) ResolveSecretSharesAccusationsMessages(
 	for _, message := range messages {
 		accuserID := message.senderID
 		for accusedID, revealedAccuserPrivateKey := range message.accusedMembersKeys {
-			if sjm.ID == accusedID || int(accusedID) > sjm.group.GroupSize() {
+			isAccusedValid := accusedID > 0 && int(accusedID) <= sjm.group.GroupSize()
```

Comment:
can we add `id`? `isAccusedIdValid`

Reply:
Yeah, good idea but it has to be `isAccusedIDValid` then because Go linter complains about `Id` vs `ID` (:untada:)